.editorconfig

```diff
@@ -0,0 +1,31 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+trim_trailing_whitespace = true
+max_line_length = 120
+ij_visual_guides = 100,120,150
+
+[{go.mod,go.sum,*.go}]
+insert_final_newline = true
+indent_size = tab
+indent_style = tab
+tab_width = 2
+
+[Makefile]
+max_line_length = off
+insert_final_newline = true
+indent_size = tab
+indent_style = tab
+tab_width = 2
+
+[*.md]
+max_line_length = off
```

Comment:
I would have used something like 200, but why not

Reply:
I thought about enabling the limit, but it would add automatic text wrapping in the same IntelliJ editor and then I would have to add the rule "ij_wrap_on_typing = false". But I don’t want to complicate the set of rules with specific rules for a particular development environment
I thought that the linter + visual rule "ij_visual_guides" should be quite enough

Reply:
I'm fine with it 👍